test/performance/1-basic-tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+REPEAT=$1
+
+echo 'azmi - build executable'
+dotnet build src/azmi-commandline/azmi-commandline.csproj
+
+exePath=$(cd ./src/azmi-commandline/bin/Debug/netcoreapp3.0 || exit; pwd)
+PATH="$PATH:$exePath"
```

Comment:
I think you want it vice-versa.
If azmi in PATH, recent build won't by used.

Reply:
Also think of export keyword. I think should be Ok here.

Reply:
I will check on `export`